File: TuringTest/pallindrom.py
class Palindrome:
    @staticmethod
    def is_palindrome(word):
        #Please write your code here.
        x = word.lower()
        y = x[::-1]
        r = False
        if x == y:
            r = True
        return(word + ' : %s' % r)
        return None

print('Input the word : ')
word = input()
print(Palindrome.is_palindrome(word))

File: TuringTest/pallindrom.js
function isPalindrome(word)
{
  x = word.toLowerCase();
  y = ''
  // Please write your code here.
  for (var i = x.length - 1; i >= 0; i--) {
      y += x[i];
  }

  if (x == y){
    return true;
  } else {
    return false;
  }
}

var w = ['oppO', 'samsung']

for (i = 0; i < w.length; i++){
    console.log(w[i] + ' : ' + isPalindrome(w[i]))
}